tracing/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "near-tracing"
```

Comment:
We already got o11y, is there a good reason to introduce a separate crate if o11y would end up depending on this anyway?